crates/router/src/core/user/dashboard_metadata.rs

```diff
@@ -447,6 +450,11 @@ async fn insert_metadata(
             metadata
         }
         types::MetaData::ProdIntent(data) => {
+            if let Some(poc_email) = &data.poc_email {
+                let inner_poc_email = poc_email.peek().as_str();
+                common_utils::pii::Email::from_str(inner_poc_email)
```

Comment:
Import `common_utils` separately.
```suggestion
                pii::Email::from_str(inner_poc_email)
```